scripts/flail.py

```diff
@@ -12,60 +12,79 @@
 
 class Flail:
     """
-        The flaing consists in converting a file to a C header containing all the contents
-        as an uint array
+        The input file can be either an object file (e.g. a goto binary file)
+        or a list of C source file. The flailng converts the content of an input file
+        into an uint array. The resulting array will be written in a C header.
 
         Example:
 
         Step 1: Object Dump
-
             #ifndef __ESBMC_HEADERS_STDARG_H_
             #define __ESBMC_HEADERS_STDARG_H_
             ...
-                    vvv
+            is represented by the code as follows:
```

Comment:
This is a hex object, not a code.

Reply:
Aren't they the ASCII encoding of "#ifndef __ESBMC_HEADERS_STDARG_H_"?

Reply:
Fixed.